src/GoslingMetaComponent.tsx

```diff
@@ -1,72 +1,70 @@
-import React, { useRef, useState, useEffect } from 'react';
-import { GoslingComponent, type GoslingRef, type GoslingSpec } from 'gosling.js';
-import { type Datum } from 'gosling.js/dist/src/core/gosling.schema';
-import MetaTable from './MetaTable';
+import React, {useRef, useState, useEffect} from 'react';
+import {GoslingComponent, type GoslingRef, type GoslingSpec} from 'gosling.js';
+import MetaTable, {MetaTableSpec} from './MetaTable';
 import 'higlass/dist/hglib.css';
 import './index.css';
 
-interface MetaSpec {
-    type: 'table'; // just an example of a user spec
-    // ...
+export type MetaSpec = {
+    width: number;
+    height?: number;
+} & MetaTableSpec
+
+interface AlignmentType {
+    type: 'loose' | 'tight';
+    trackID: string;
```

Comment:
We can follow the convention used in Gosling projects, i.e., we always use the lower case for the second letter of a word regardless of whether it is an abbreviation or not. I think this makes it easier to remember the letter cases.

```suggestion
    trackId: string;
```